system/version/goos/activate_tok_darwin.go

```diff
@@ -0,0 +1,41 @@
+package goos
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ironstar-io/tokaido/constants"
+	"github.com/ironstar-io/tokaido/system/fs"
+	"github.com/ironstar-io/tokaido/utils"
+)
+
+// ActivateSavedVersion - Copies the specified version (which may be downloaded previously) into /usr/local/bin on macOS
+func ActivateSavedVersion(version string) bool {
+	// Check that the version is downloaded already
+	p := filepath.Join(fs.HomeDir(), constants.BaseInstallPathDarwin, version, "tok")
+	if fs.CheckExists(p) != true {
+		utils.DebugString("Tokaido version [" + version + "] was not found in ~/.tok/bin, downloading a new copy...")
+
+		_, err := DownloadTokBinary(version)
+		if err != nil {
+			fmt.Println("Unexpected error downloading that version: " + err.Error())
+			os.Exit(1)
+		}
+	}
+
+	// Remove any existing copy of Tokaido at /usr/local/bin/tok
+	fs.Remove(constants.ActiveBinaryPathDarwin)
+
+	// Copy the specified version to /usr/local/bin/tok
+	fs.Copy(p, constants.ActiveBinaryPathDarwin)
+
+	// Make sure the version is executable
+	err := os.Chmod(constants.ActiveBinaryPathDarwin, 0777)
```

Comment:
`777` or `755`?

Reply:
I rationalised this because we won't know what permissions they have on that file (they may not be the owner). 770 would probably be fine, but giving every user on a workstation access to run the tokaido binary isn't a security risk. This is the safest option.